test/e2e/memcached_test.go

```diff
@@ -479,23 +481,31 @@ func memcachedScaleTest(t *testing.T, f *framework.Framework, ctx *framework.Tes
 		t.Fatalf("Failed to unmarshal memcached CR: (%v)", err)
 	}
 	obj.SetNamespace(namespace)
+
 	err = f.Client.Get(context.TODO(), types.NamespacedName{Name: obj.GetName(), Namespace: obj.GetNamespace()}, &obj)
 	if err != nil {
 		return fmt.Errorf("failed to get memcached object: %s", err)
 	}
-	// update memcached CR size to 4
-	spec, ok := obj.Object["spec"].(map[string]interface{})
-	if !ok {
-		return errors.New("memcached object missing spec field")
-	}
-	spec["size"] = 4
-	err = f.Client.Update(context.TODO(), &obj)
+
+	err = retry.RetryOnConflict(retry.DefaultBackoff, func() error {
+		// update memcached CR size to `toReplicas` replicas
+		spec, ok := obj.Object["spec"].(map[string]interface{})
+		if !ok {
+			return errors.New("memcached object missing spec field")
+		}
+		spec["size"] = toReplicas
+		t.Logf("Attempting memcached CR update, resourceVersion: %s", obj.GetResourceVersion())
+		return f.Client.Update(context.TODO(), &obj)
+	})
 	if err != nil {
-		return err
+		return fmt.Errorf("could not update memcached CR: %v", err)
```

Comment:
this is a nit: but if `CR: %v` I would put the NamespacedName in for the CR and then add a second `%v` to wrap the error WDYT?